tests/unit/helpers/sort-option-display-test.js

```diff
@@ -0,0 +1,12 @@
+
+import { sortOptionDisplay } from 'dummy/helpers/sort-option-display';
+import { module, test } from 'qunit';
+
+module('Unit | Helper | sort option display');
+
+test('returns corresponding display text', function(assert) {
```

Comment:
I just wanted to say thanks again for your work adding tests to this code- both the new and inherited parts. I know there will be a ways to go yet (this is a very complex feature!).... but this will make a big step forward. Your diligence and care is really appreciated.